english/marshmallow.md

```diff
@@ -1192,3 +1192,5 @@ Maybe this is a [dream](dream/dream.md)?
 
 Or maybe you love [pugs] (https://www.youtube.com/watch?v=3pPR7vk-e3Y)?
 
+Have you ever heard any unusual sounds when you're alone? They're not what you think. Get to know them [here](https://github.com/Drithin/create-your-own-adventure/tree/master/english/blackAndwhite/Chapter%201)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.